cql/src/main/java/org/apache/cassandra/stargate/transport/internal/CBUtil.java

```diff
@@ -367,7 +391,7 @@ public static ByteBuffer readBoundValue(ByteBuf cb, ProtocolVersion protocolVers
           ProtocolVersion.V4)) // backward compatibility for pre-version 4
       return null;
       if (length == -1) return null;
-      else if (length == -2) return ByteBufferUtil.UNSET_BYTE_BUFFER;
+      else if (length == -2) return unsetValue;
```

Comment:
minor: Maybe we should assert that `unsetValue` is set?

Reply:
Yes. Meant to but forgot, sorry.